### PR TITLE
Add NZXT kraken X (X42, X52, X62 or X72) Support

### LINKED ISSUE #1095: Support for NZXT Kraken X2
I've seen that recently support for some NZXT devices has been added. I tried to control my X62 but it didn't show up on this program.
@brokenmass could you add support for these devices? If you need someone to test I would be glad to help!

### PR DESCRIPTION
**Pull request by mgth**
Support tested on Kraken X62 Firmware 6.2 should fix #1095

### CODE REVIEW

**Comment by PhyxionNL** [comment]
Can this be integrated into KrakenV3? Just glancing over it, and it seems a lot of code is quite similar.

File: LibreHardwareMonitorLib/Hardware/Controller/Nzxt/KrakenV2.cs [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+    public override void Update()
+    {
+        try
+        {
+            using HidStream stream = _device.Open();
+
+            stream.Read(_rawData);
+
+            // if not 0x04, it is not temperature data
+            if (_rawData[0] != 0x04) return;
```
Comment (PhyxionNL) [inline LibreHardwareMonitorLib/Hardware/Controller/Nzxt/KrakenV2.cs:156]: Can you reformat this document and follow code style (e.g. return on new line).

File: LibreHardwareMonitorLib/Hardware/Motherboard/Identification.cs [hunk 2]
```diff
@@ -411,6 +411,8 @@ public static Model GetModel(string name)
                 return Model.B660M_DS3H_AX_DDR4;
             case var _ when name.Equals("ROG STRIX Z790-I GAMING WIFI", StringComparison.OrdinalIgnoreCase):
                 return Model.ROG_STRIX_Z790_I_GAMING_WIFI;
+            case var _ when name.Equals("MPG X570 GAMING PLUS (MS-7C37)",StringComparison.OrdinalIgnoreCase):
```
Comment (PhyxionNL) [inline LibreHardwareMonitorLib/Hardware/Motherboard/Identification.cs:414]: Please move this added motherboard to a separate PR.

**Review by PhyxionNL — CHANGES_REQUESTED** [review]

File: LibreHardwareMonitorLib/Hardware/Motherboard/Identification.cs (hunk 2 above)
Comment (mgth) [reply LibreHardwareMonitorLib/Hardware/Motherboard/Identification.cs:414]: oups, sorry about that it was not supposed to be here

**Comment by mgth** [comment]
sorry did PR on wrong branch, done a new one : #1143
